### ISSUE TITLE
This is awesome! - any updates?

### ISSUE THREAD

**Issue by ihrapsa** [FernandoGarcia/ParkingSensor#1]
I'm astonished that there are no after market parking sensors that can communicate with android headunits. The Rusonic WiSec seems to be doing just that but I can't find it anywhere to buy. There's also  FenSense but they seem to be pretty badly reviewed. These chinese kits are everywhere and it would be pitty not to hack them 😅

Thank you for publishing your work!

Are there any updates on this regarding the android app development?
I'd like to test and help with feedback! (would've helped with more but my skills end there unfortunately)

Jonah

**Comment by FernandoGarcia** [comment]
Hi!

For now I haven't found any Android developer interested in this project.

Best regards.

**Comment by closb** [comment]
Hey bud, I'm a developer. Haven't worked on Android but I'm also looking at doing same thing for my 2003 Jaguar X-type project car.

I just started doing research on this and came across your post on xdr forum. That said, I'm interested in collaborating, I'll handle software and you keep tinkering with hardware?

LMK.

**Comment by FernandoGarcia** [comment]
Hi!
  

  
I'll work on ESP32 code.
  

  
I think that a custom board is not needed for this project since the wiring is very simple but eventually I can make a board to join the ESP and the step down.
  

  
The problem with a custom board is the size since each parking sensor has a different free space on enclosure.
  

  
Let me now if I should create a repository for Android stuffs.
  

  
You can send me an email at info@ ferduino.com with your skype so we can talk directly.
  

  
Best regards.

**Comment by closb** [comment]
Sounds good, will do!

**Comment by FernandoGarcia** [comment]
Hi @closb!

Do you have the hardware to build the system? 

I'm waiting for your email.

Best regards.

**Comment by closb** [comment]
> Hi @closb!
> 
> Do you have the hardware to build the system? 
> 
> I'm waiting for your email.
> 
> Best regards.

Hey bud,

Sorry I just finished moving last weekend and am still unboxing and getting things set up.

That said, ATT finally gave me my block of IPs so I can start setting up my servers.

Conversely, my parking sensors arrived this weekend finally. They're an off brand so I don't have specifics yet.

Nonetheless, if all goes well, I'm hoping to start tinkering this weekend.

**Comment by closb** [comment]
Hope all is well. I'm working on an issue with my transmission range sensor so I'm behind on rest.

Nonetheless, I started looking at the docs for Andriod auto.

When I looked into this originally, I imagined creating a bird's-eye view UI so I'll be adding 4 cameras to the mix.

I found the docs on how to stitch the images together so I'd like to continue on this path.

Feel free to drop me a message at barberenac@outlook.com so we can connect and start planning.
